storybook/src/components/Column/Column.stories.tsx

```diff
@@ -32,34 +54,57 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
+export const Alignment: Story = {
+  args: {
+    align: 'center',
+  },
+  argTypes: {
+    alignHorizontal: { table: { disable: true } },
+    as: { table: { disable: true } },
+    gap: { table: { disable: true } },
+  },
```

Comment:
Why did you disable these controls here? We generally allow all controls for all stories.

Reply:
They’re all back.